api-docs/api-reference/methods/extensions/get-list-extensions-v2.0.rst

```diff
@@ -8,9 +8,9 @@ List extensions
     GET //v2.0/extensions
 
 This operations lists the Identity API extensions that are available in the
-Rackspace Cloud Identity  service. For more information, see the
-:ref:`Extensions <extensions-ovw>` section of the  Rackspace Cloud Identity
-service Developer Guide.
+Rackspace Identity  service. For more information, see the
```

Comment:
I think there might be an extra space before "service."

Reply:
Fixed